### ISSUE TITLE
【framepack】 I found that LoRA trained on longer videos will be blurry.

### ISSUE THREAD

**Issue by xianienie** [kohya-ss/musubi-tuner#387]
I found that when using framepack to train longer videos, such as those with more than 8 caches, the generated videos tend to be blurry. I tried adjusting many parameters, but this seems unrelated to flow shift and network dim. Additionally, I found that when training longer videos, the learning rate needs to be lowered for the loss to decrease. Therefore, I speculate that this is because the loss is calculated with the entire video during computation, and longer videos involve more content in the calculation, leading to a failure to fit.Especially in high-frequency areas, this leads to not learning the details, which then causes blurriness.

**Comment by Password1314** [comment]
I'd like to ask, each of my training videos has about 180 frames. Is a learning rate of 2e-4 reasonable?
